.github/workflows/build.yml

```diff
@@ -2,19 +2,19 @@ name: build
 on: [push, pull_request]
 jobs:
   linters:
-    name: linters
+    name: linters / on ${{ matrix.os }}
```

Comment:
suggest lint and vendor check

Reply:
Moved to #639

Reply:
s/linters/lint and vendor check/
see line 34 below